lib/suspenders/app_builder.rb

```diff
@@ -197,6 +197,11 @@ def enable_database_cleaner
       copy_file 'database_cleaner_rspec.rb', 'spec/support/database_cleaner.rb'
     end
 
+    def provide_shoulda_matchers_config
+      copy_file 'shoulda_matchers_config_rspec.rb',
+        'spec/support/shoulda_matchers_config.rb'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.